src/main/java/de/retest/recheck/persistence/JUnitTestbasedNamingStrategy.java

```diff
@@ -0,0 +1,39 @@
+package de.retest.recheck.persistence;
+
+import de.retest.recheck.TestCaseFinder;
+
+/**
+ * The default implementation that derives test suite and test case name from the test class and test method, working
+ * for both JUnit and TestNG.
+ */
+public class JUnitTestbasedNamingStrategy implements NamingStrategy {
```

Comment:
```suggestion
public class JUnitTestBasedNamingStrategy implements NamingStrategy {
```

Reply:
I had this conversation with Daniel once. We agreed to accept whatever comes out of https://capitalizemytitle.com/camel-case/ ... in our case for "JUnit-test-based naming strategy". After seeing that, I shortened it to "JUnit-based naming strategy"...